src/style.css

```diff
@@ -0,0 +1,97 @@
+@import "./cardStyles/__normalise.css";
+@import "./cardStyles/__variables.css";
+@import "./cardStyles/__feedbackStars.css";
+
+.home-page {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+}
+
+.card {
+  display: block;
+  width: var(--card-width);
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px ;
```

Comment:
https://codeguide.academy/html-css.html#css-order, fix your css order everywhere, according to this guide

Reply:
Good advice. Code adjusted acordingly.